animated.md

```diff
@@ -1,9 +1,6 @@
 ### Movie list
```

Comment:
"L" should be capital.

Reply:
I have made necessary changes @elinaDangol .